doc/getting_started/index.rst

```diff
@@ -7,16 +7,19 @@ After reading the :ref:`introduction`, use this guide to get started
 using ACRN in a reference setup.  We'll show how to set up your
 development and target hardware, and then how to boot up the ACRN
 hypervisor and the `Clear Linux`_ Service OS and Guest OS on the Intel
-|reg| NUC.
+|reg| (EFI) platform.
```

Comment:
If you remove the word NUC, then the Intel should not have a registered trademark, so delete ``|reg|``